org.aposin.licensescout.core/pom.xml

```diff
@@ -114,14 +126,14 @@
 				</executions>
 			</plugin>
 
+<!--
+NOTE: disabled because of CI build error 
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-deploy-plugin</artifactId>
-				<configuration>
-					<skip>false</skip>
-				</configuration>
+				<artifactId>maven-source-plugin</artifactId>
 			</plugin>
-
+ -->
```

Comment:
This is disabled now, but it was not really in the project. Any reason to include it commented out?

Reply:
Yes, there is a reason. Once we do public releases on Maven central, it is common and good practice to also provide the source code bundled as a JAR (which is needed for seamless debugging). So we should find a way to include this. One possibility is to have a Maven profile used only for release (but not for the CI build, where bundling the source code does not make sense anyway).

However, this should be further investigated in the context of figuring out how to do the release to Maven central - therefore no final solution at the moment.